dotCMS/src/main/webapp/dotcms-webcomponents/p-5ncsofev.system.entry.js

```diff
@@ -0,0 +1 @@
0]?a[1]:void 0,done:true}}};System.register(["./p-aa15c74d.system.js"],(function(t){"use strict";var e,n,r,i,o;return{setters:[function(t){e=t.r;n=t.c;r=t.h;i=t.H;o=t.g}],execute:function(){var a=function(t){return t&&typeof t==="string"?t.split(","):[]};var c=function(t,e){if(t&&t.length&&e&&typeof e==="string"){return t.filter((function(t){var n=t.data.inode;return e.split(",").includes(n)})).map((function(t){var e=t.data;return e}))}return[]};var u=t("dot_card_view",function(){function t(t){e(this,t);this.items=[];this.selection=[];this.selected=n(this,"selected",7);this.cardClick=n(this,"cardClick",7)}t.prototype.watchItems=function(t){this.selection=c(t,this.value)};t.prototype.watchValue=function(t){this.selection=c(this.items,t)};t.prototype.getValue=function(){return __awaiter(this,void 0,void 0,(function(){return __generator(this,(function(t){return[2,this.selection]}))}))};t.prototype.clearValue=function(){return __awaiter(this,void 0,void 0,(function(){var t;return __generator(this,(function(e){this.value="";t=this.getCards();t.forEach((function(t){t.checked=false}));return[2]}))}))};t.prototype.componentDidLoad=function(){this.selection=c(this.items,this.value);this.cards=this.getCards()};t.prototype.clearMenu=function(){this.cards.forEach((function(t){t.hideMenu()}))};t.prototype.render=function(){var t=this;var e=a(this.value);return r(i,null,this.items.map((function(n){return r("dot-card-contentlet",{onContextMenu:function(e){return __awaiter(t,void 0,void 0,(function(){var t;return __generator(this,(function(n){e.preventDefault();t=e.target;this.clearMenu();t.showMenu(e.x,e.y);return[2]}))}))},onClick:function(){t.clearMenu();t.cardClick.emit(n.data)},key:n.data.inode,checked:e.includes(n.data.inode),onCheckboxChange:function(e){var r=e.detail,i=r.originalTarget,o=r.shiftKey;var a=false;if(o&&i.checked){t.cards.forEach((function(e){if(e===i||e===t.lastChecked){a=!a}if(a){e.checked=true}}))}t.lastChecked=i;t.setValue(i,n.data)},item:n})})))};t.prototype.setValue=function(t,e){if(t.checked){this.selection.push(e)}else{this.selection=this.selection.filter((function(t){var n=t.identifier;return n!==e.identifier}))}this.value=this.selection.map((function(t){var e=t.inode;return e})).join(",");this.selected.emit(this.selection)};t.prototype.getCards=function(){return this.el.shadowRoot.querySelectorAll("dot-card-contentlet")};Object.defineProperty(t.prototype,"el",{get:function(){return o(this)},enumerable:true,configurable:true});Object.defineProperty(t,"watchers",{get:function(){return{items:["watchItems"],value:["watchValue"]}},enumerable:true,configurable:true});Object.defineProperty(t,"style",{get:function(){return":host{display:grid;grid-template-columns:repeat(auto-fill,minmax(220px,1fr));grid-gap:var(--basic-padding-2)}dot-card-contentlet{height:100%}dot-card-contentlet:before{content:\"\";display:inline-block;-ms-flex:0 0 0px;flex:0 0 0;height:0;padding-bottom:calc(100%)}"},enumerable:true,configurable:true});return t}())}}}));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A 'return', 'break', 'continue', or 'throw' statement should be the last in a block.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5219792)